code/game/objects/items/storage/backpack.dm

```diff
@@ -155,6 +155,12 @@
 	. = ..()
 	new /obj/item/storage/wallet/random( src )
 
+
+/obj/item/storage/backpack/satchel/som
+	name = "mining satchel"
+	desc = "A satchel with origins dating back to the mining colonies."
+	icon_state = "som_lightpack" //todo:new sprite
```

Comment:
hi

Reply:
I did actually make a sprite, helps if I added it to the pr :^)